app/views/install/compose.phtml

```diff
@@ -86,7 +86,7 @@ $image = $this->getParam('image', '');
       - _APP_OPTIONS_ABUSE
       - _APP_OPTIONS_ROUTER_PROTECTION
       - _APP_OPTIONS_FORCE_HTTPS
-      - _APP_OPTIONS_FUNCTIONS_FORCE_HTTPS
+      - _APP_OPTIONS_COMPUTE_FORCE_HTTPS
```

Comment:
This shouldn't have anything to do with compute. Sounds more like a proxy/router/edge feature.

Reply:
Renamed to `_APP_OPTIONS_ROUTER_FORCE_HTTPS` (router, because we already got `_APP_OPTIONS_ROUTER_PROTECTION`)